src/screens/tabs/NFTs/components/AddCollection/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { t } from 'i18next';
+import React, { useRef, useState } from 'react';
+import { Modalize } from 'react-native-modalize';
+
+import ScrollableButton from '@/components/buttons/ScrollableButton';
+import { ActionButton, Header, Modal, Text } from '@/components/common';
+import { Nullable } from '@/interfaces/general';
+import { CollectionInfo } from '@/interfaces/keyring';
+
+import CustomCollection from '../CustomCollection';
+import ReviewCollection from '../ReviewCollection';
+import styles from './styles';
+
+interface Props {
+  scrollPosition: number;
+}
+
+function AddCollection({ scrollPosition }: Props) {
+  const modalRef = useRef<Modalize>(null);
+  const [selectedCollection, setSelectedCollection] =
+    useState<Nullable<CollectionInfo>>(null);
```

Comment:
same here with nullable
```suggestion
    useState<CollectionInfo>();
```

and `const cleanState = () => setSelectedCollection(undefined);`

Reply:
done!